crates/uv/tests/lock.rs

```diff
@@ -981,7 +979,7 @@ fn lock_conditional_dependency_extra() -> Result<()> {
         name = "project"
         version = "0.1.0"
         requires-python = ">=3.7"
-        dependencies = ["requests", "requests[socks] ; python_version < '3.8'"]
+        dependencies = ["requests", "requests[socks] ; python_version < '3.10'"]
```

Comment:
Out of curiosity, why the change in Python version here?

Reply:
I think it was because we typically test against 3.12 and 3.8. (This isn't required, it's just the common versions we use in these tests.) And so if I left it as-is, I would've had to install on 3.7 to hit the "exclusion" case. So it was just a bit easier to raise to 3.10, so that 3.12 is above and 3.8 is below.